jaspTTests.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
+
+RestoreWorkspace: Default
+SaveWorkspace: Default
+AlwaysSaveHistory: Default
+
+EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
+NumSpacesForTab: 2
+Encoding: UTF-8
+
+RnwWeave: Sweave
+LaTeX: pdfLaTeX
+
```

Comment:
```suggestion

AutoAppendNewline: Yes
StripTrailingWhitespace: Yes

```
that would ensure there are no unnessesary files changed due to trailing whitespace and Windows vs unix line endings.